test/unit/algolia/index.js

```diff
@@ -0,0 +1,81 @@
+const byteof = require('byteof');
+const {expect} = require('chai');
+
+const {
+  chunkAlgolia,
+  maxChunkSizeInBytes,
+  maxItemSizeInBytes,
+  trimBytes,
+} = require('../../../algolia');
+
+/**
+ * Very simple to mock dummy object, only accepts even byte sizes.
```

Comment:
Replace "dummy" with "temp" or "placeholder" to avoid the ableist usage. See: https://developers.google.com/style/word-list#dummy-variable